benchmarks/jmh/src/jmh/kotlin/com/linecorp/armeria/internal/common/kotlin/AnnotatedServiceFlowBenchmark.kt

```diff
@@ -37,32 +37,32 @@ import java.util.stream.IntStream
 @Warmup(iterations = 1)
 @Suppress("unused")
 open class AnnotatedServiceFlowBenchmark {
-
     lateinit var server: Server
     lateinit var client: WebClient
 
     @Setup
     open fun setup() {
-        server = Server.builder()
-            .annotatedService(
-                "/benchmark",
-                object {
-                    @Get("/flow")
-                    @ProducesJsonSequences
-                    fun flowBm(): Flow<String> = flow {
-                        (0 until 1000).forEach {
-                            emit("$it")
-                        }
-                    }
+        server =
```

Comment:
I recommend enabling `Hide whitespace changes` option to clear massive changes by ktlint.

Reply:
Genuinely, I don't understand why ktlint keeps changing the default format for every major update. 😵‍💫🤯